CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Fixed
 * Bump JS parser up to parse ES2017 (ES8).
+* Correctly parse shorthand property definitions.
```

Comment:
This isn't clear to a reader that doesn't already know about the bug I don't think. What about this:

```md
* [Polymer] Property descriptors are allowed to be just a type name, like `value: String`.
```